src/components/popover/popover-trigger/popover-trigger.stories.ts

```diff
@@ -77,15 +77,14 @@ const popover = (): TemplateResult => html`
     </sbb-title>
     <p class="sbb-text-s" style="margin: 0;">
       Some content.
-      <sbb-link
+      <sbb-block-link
```

Comment:
should we not adapt the import to the block link? the current way, all link variants would be loaded. Maybe also a question for other stories?

Reply:
I thought about it and I left the broader import on purpose. 
The thought process is:
- In storybook, we don't have to super-optimize the bundle size since it's just a showcase environment.
- With Turbosnap enabled, it's essential that the component is imported correctly to enable Turbosnap to navigate the dependency graph and decide which snapshot to take. 

For example, yesterday Davide noticed that the snapshots of the navigation-section were not taken after a change he made. The reason was that the section was not imported in stories, but still worked :|

Essentially, the reason I would keep it like this is "Better safe than sorry" :D

Reply:
I agree with the not super optimized storybook, but for turbosnap it should also be ok if you only import the block link, not? Davide's case is another case, where it was really forgotten I think.

Reply:
Yes, my point was that the specific import is just more prone to error (since the various links are easily interchangeable)